pkg/capabilities/consensus/ocr3/datafeeds/feeds_aggregator.go

```diff
@@ -21,8 +21,9 @@ type aggregatorConfig struct {
 }
 
 type feedConfig struct {
-	Deviation decimal.Decimal
-	Heartbeat int
+	Deviation       decimal.Decimal `mapstructure:"-"`
+	Heartbeat       int
+	DeviationString string `mapstructure:"deviation"`
```

Comment:
What's the problem with Decimal? Values support it...

Reply:
Yaml doesn't :/